### PR TITLE
Finishing touch

### PR DESCRIPTION
**Pull request by anastygnome**
This commit ensures the Slots are filtered and cleans  a few things up

### CODE REVIEW

File: edtLHD/src/main/java/fr/univtln/lhd/facade/Schedule.java [hunk 1]
```diff
@@ -33,13 +32,11 @@
  */
 @Slf4j
 public class Schedule implements Observable {
-    private static Map<String, List<Observer>> observerMap = new HashMap<>();
+    public static final String SLOT_EVENT = "slotEvent";
```
Comment (Theo-Hafsaoui) [inline edtLHD/src/main/java/fr/univtln/lhd/facade/Schedule.java:35]: are you **absolutely** sure about that

Comment (anastygnome) [reply edtLHD/src/main/java/fr/univtln/lhd/facade/Schedule.java:35]: this variable was unused, it got linted

File: edtLHD/src/main/java/fr/univtln/lhd/facade/Schedule.java [hunk 2]
```diff
@@ -33,13 +32,11 @@
...
     private static final String ERR_MSG_DELETE_UPDATE = "this entity doesn't seem persisted";
 
     private static final Schedule schedule = new Schedule();
-
-    public static final String SLOT_EVENT = "slotEvent";
-
+    private static Map<String, List<Observer>> observerMap = new HashMap<>();
```
Comment (anastygnome) [inline edtLHD/src/main/java/fr/univtln/lhd/facade/Schedule.java:39]: @Theo-Hafsaoui  there it is

Comment (Theo-Hafsaoui) [reply edtLHD/src/main/java/fr/univtln/lhd/facade/Schedule.java:39]: i see

**Pull request merged**